pkg/cluster/internal/kubeadm/config.go

```diff
@@ -169,6 +181,11 @@ kind: JoinConfiguration
 # no-op entry that exists soley so it can be patched
 apiVersion: kubelet.config.k8s.io/v1beta1
 kind: KubeletConfiguration
+evictionHard:
+  memory.available: "1"
```

Comment:
cc @neolit123 @dashpole for some reason cluster up fails when this is `memory.available: "0"`. I can't find any reason that should be invalid yet though.

Reply:
i cannot find where in the source 0 would be a special case.
also this seems undocumented.

Reply:
we're now not setting this key, but this is still... curious